docs/reference/src/quickstart/main_tiles_from_js.js

```diff
@@ -4,18 +4,20 @@
 // ANCHOR: main
 // main.js
 import * as slint from "slint-ui";
-let ui = slint.loadFile("./ui/app-window.slint");
-let mainWindow = new ui.MainWindow();
+const ui = slint.loadFile(new URL("./ui/app-window.slint", import.meta.url));
+const mainWindow = new ui.MainWindow();
 
-let initial_tiles = mainWindow.memory_tiles;
-let tiles = initial_tiles.concat(initial_tiles.map((tile) => Object.assign({}, tile)));
+const initial_tiles = mainWindow.memory_tiles;
```

Comment:
The change of line count here (and in the other .js file(s)) breaks the quick start guide that extracts from those files by line range. Please adjust the quick start guide to match this.